libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -65,6 +65,10 @@ const AP_Param::GroupInfo AP_DDS_Client::var_info[] {
     // @User: Standard
     AP_GROUPINFO("_PORT", 2, AP_DDS_Client, udp.port, 2019),
 
+    // @Group: _IP
```

Comment:
Is there a way to add a description
"IP address of the DDS Agent"

Reply:
I don't think so.

Note, the _OPT docs are broken. Docs say "DDS_UDP_PORT" but code is "DDS_PORT". I'm in favor of removing the "_UDP_" part and have a different param control TCP/UDP server/client options